include/ros1_bridge/factory.hpp

```diff
@@ -45,9 +45,10 @@ class Factory : public FactoryInterface
   create_ros1_publisher(
     ros::NodeHandle node,
     const std::string & topic_name,
-    size_t queue_size)
+    size_t queue_size,
+    bool latch)
```

Comment:
Why does the API not match the interface - with the `latch` argument being optional?

Same below for the other optional arguments.

Reply:
fixed, thanks